x-pack/test/reporting_api_integration/services.ts

```diff
@@ -56,7 +59,13 @@ export function ReportingAPIProvider({ getService }: FtrProviderContext) {
             .get(downloadReportPath)
             .responseType('blob')
             .set('kbn-xsrf', 'xxx')) as any;
-          log.debug(`Report at path ${downloadReportPath} returned code ${response.statusCode}`);
+          if (response.statusCode === 503) {
+            log.debug(`Report at path ${downloadReportPath} is pending`);
+          } else if (response.statusCode === 200) {
+            log.debug(`Report at path ${downloadReportPath} is complete`);
+          } else {
+            log.debug(`Report at path ${downloadReportPath} returned code ${response.statusCode}`);
+          }
```

Comment:
I added this change because it makes it a bit easier to see what is going on while watching the logs.